package.json

```diff
@@ -7,10 +7,12 @@
   "license": "MIT",
   "repository": "https://github.com/Axinom/drm-quick-start",
   "dependencies": {
+    "commander": "^2.9.0",
```

Comment:
I wonder if the new entries should be devDependencies instead? I always forget the exact logic - just something that should be double checked... maybe it does not even matter.